content/batch/artifact.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Configure Artifact Repository"
+date: 2018-11-18T00:00:00-05:00
+weight: 50
+draft: false
+---
+
+### Configure Artifact Repository
+
+Argo uses an artifact repository to pass data between jobs in a workflow, known as artifacts. Amazon S3 can be used as an artifact repository.
+
+Let's create a S3 bucket using the AWS Service Operator from earlier.
+
+```bash
+ACCOUNT_ID=$(aws sts get-caller-identity --output text --query Account)
+
+cat <<EoF > ~/environment/batch-artifact-repository.yaml
+---
+apiVersion: operator.aws/v1alpha1
+kind: S3Bucket
+metadata:
+  name: batch-artifact-repository-${ACCOUNT_ID}
+spec:
+  versioning: false
+EoF
+
+kubectl apply -f ~/environment/batch-artifact-repository.yaml
+```
+
+{{%expand "Create S3 bucket using AWS CLI instead" %}}
```

Comment:
Is there a reason you would ask to create S3 bucket using AWS CLI?

Reply:
I had trouble with the service operator.

Reply:
Removed service operator and switched to AWS CLI. Will switch to service operator at a later date.